front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import cx from 'classnames';
+
+import type {
+  PathPropertiesFormatted,
+  SimulationResponseSuccess,
+} from 'applications/operationalStudies/types';
+import type { TrainScheduleResult } from 'common/api/generatedEditoastApi';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainScheduleV2/types';
+import type { PathStep } from 'reducers/osrdconf/types';
+import { NO_BREAK_SPACE } from 'utils/strings';
+
+import useOutputTableData from './hooks/useOutputTableData';
+import TimesStops from './TimesStops';
+import { TableType, type PathWaypointRow } from './types';
+
+type TimesStopsOutputProps = {
+  simulatedTrain: SimulationResponseSuccess;
+  pathProperties: PathPropertiesFormatted;
+  operationalPoints: OperationalPointWithTimeAndSpeed[];
+  selectedTrainSchedule: TrainScheduleResult;
+  startTime: string;
+  pathSteps: PathStep[];
+  pathLength?: number;
+};
+
+const TimesStopsOutput = ({
+  simulatedTrain,
+  pathProperties,
+  operationalPoints,
+  selectedTrainSchedule,
+  startTime,
+  pathSteps,
+  pathLength,
+}: TimesStopsOutputProps) => {
+  const enrichedOperationalPoints = useOutputTableData(
+    simulatedTrain,
+    pathProperties,
+    operationalPoints,
+    selectedTrainSchedule,
+    startTime,
+    pathLength
+  );
+  return (
+    <TimesStops
+      allWaypoints={enrichedOperationalPoints}
+      pathSteps={pathSteps}
+      startTime={startTime}
+      tableType={TableType.Output}
+      cellClassName={({ rowData: rowData_ }) => {
+        const rowData = rowData_ as PathWaypointRow;
+        const arrivalScheduleNotRespected = rowData.arrival
+          ? rowData.calculatedArrival !== rowData.arrival
+          : false;
+        const negativeDiffMargins = Number(rowData.diffMargins?.split(NO_BREAK_SPACE)[0]) < 0;
+        return cx({
+          'warning-schedule': arrivalScheduleNotRespected,
+          'warning-margin': negativeDiffMargins,
+        });
+      }}
+    />
```

Comment:
Small improvement: currently, if I select a new train, I can still see for a moment the results of the previous train and there is nothing to notify me that the data has finished to load. Maybe we can add a loader like this (you can reuse formattedOpPointsLoading in SimulationResultsV2)
```suggestion
    // Prevent the screen from resizing during loading
    <div style={{ height: '600px' }}>
      {(dataIsLoading && <Loader />) || (
        <TimesStops
          allWaypoints={enrichedOperationalPoints}
          pathSteps={pathSteps}
          startTime={startTime}
          tableType={TableType.Output}
          cellClassName={({ rowData: rowData_ }) => {
            const rowData = rowData_ as PathWaypointRow;
            const arrivalScheduleNotRespected = rowData.arrival
              ? rowData.calculatedArrival !== rowData.arrival
              : false;
            const negativeDiffMargins = Number(rowData.diffMargins?.split(NO_BREAK_SPACE)[0]) < 0;
            return cx({
              'warning-schedule': arrivalScheduleNotRespected,
              'warning-margin': negativeDiffMargins,
            });
          }}
        />
      )}
    </div>
```

Reply:
done